standards.md

```diff
@@ -149,6 +169,20 @@ requires developers to follow more links rather than just showing
 what we want.
 ```
 
+## Small Pull Requests
+
+* Small pull requests that make a single, self-contained change are easier to review and easier to roll back or backport.
+  * If you can think of a way to break up a pull request into smaller changes, do it
+* Tests and documentation (where applicable) should always be part of the same pull request when introducing a change
+* Use one commit per PR unless there is a strong reason not to
+  * For multiple commits, ensure each makes sense without the context of the others.
+* Refactoring should be merged before bug fixes and features
+  * i.e. if you refactor as part of implementing something, commit it and merge it before merging the change
```

Comment:
suggest: "if you refactor as a prerequisite that will simplify or enable your new changes, commit and merge the refactor before merging your new changes